apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.js

```diff
@@ -120,10 +120,33 @@ async function processDeleteFile(req, res) {
   await deleteFile(file.id, res.locals.authn_user.user_id);
 }
 
+/**
+ * This is used to conditionally display/permit a shortcut to delete the
+ * assessment instance. Usually, the only way to delete an assessment instance
+ * is from the "Students" tab of an assessment. However, when one is iterating
```

Comment:
```suggestion
 * is from the "Students" tab of an assessment. However, when a staff member is iterating
```
will require some reformatting.

Reply:
Done in 23e6b29.